src/mpid/ch4/shm/posix/eager/fbox/fbox_init.h

```diff
@@ -79,7 +79,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_POSIX_eager_init(int rank, int size)
     MPIR_ERR_CHECK(mpi_errno);
 
     /* Actually allocate the segment and assign regions to the pointers */
-    mpi_errno = MPIDU_shm_seg_commit(&MPIDI_POSIX_eager_fbox_control_global.memory,
+    mpi_errno = MPIDU_shm_seg_commit(&MPIDI_POSIX_eager_fbox_control_global.shm_ptr,
```

Comment:
`MPIDU_shm_seg_commit` is gone. Need clean up the commits.

Reply:
No `MPIDU_shm_seg_commit` is remove in commit 646007e75a3352a44e873e9a97c7a272aa6b3f87